AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -414,48 +434,129 @@ public async Task<ActionResult> GetHearingById(Guid hearingId)
+                    if (e.StatusCode == (int) HttpStatusCode.BadRequest) return BadRequest(e.Response);
+                    if (e.StatusCode == (int) HttpStatusCode.NotFound) return NotFound(e.Response);
                     return BadRequest(e);
                 }
+
                 return BadRequest(ex.Message);
             }
         }
 
+        /// <summary>
+        ///     Get the conference status.
+        /// </summary>
+        /// <param name="hearingId">The hearing id</param>
+        /// <returns>Success status</returns>
+        [HttpGet("{hearingId}/conference-status")]
+        [SwaggerOperation(OperationId = "GetHearingConferenceStatus")]
+        [ProducesResponseType(typeof(UpdateBookingStatusResponse), (int) HttpStatusCode.OK)]
+        [ProducesResponseType((int) HttpStatusCode.NotFound)]
+        [ProducesResponseType((int) HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> GetHearingConferenceStatus(Guid hearingId)
+        {
+            var errorMessage =
+                $"Failed to get the conference from video api, possibly the conference was not created or the kinly meeting room is null - hearingId: {hearingId}";
+            try
+            {
+                _logger.LogDebug($"Hearing {hearingId} is confirmed. Polling for Conference in VideoApi");
+                var conferenceDetailsResponse =
+                    await _conferenceDetailsService.GetConferenceDetailsByHearingId(hearingId);
+                _logger.LogInformation($"Found conference for hearing {hearingId}");
+                if (conferenceDetailsResponse.HasValidMeetingRoom())
+                {
+                    return Ok(new UpdateBookingStatusResponse
+                    {
+                        Success = true,
+                        TelephoneConferenceId = conferenceDetailsResponse.MeetingRoom.TelephoneConferenceId
+                    });
+                }
+                else
+                {
+                    return Ok(new UpdateBookingStatusResponse {Success = false, Message = errorMessage});
+                }
+            }
+            catch (VideoApiException ex)
+            {
+                _logger.LogError(ex, "Failed to confirm a hearing. {ErrorMessage}", errorMessage);
+                _logger.LogError("There was an unknown error for hearing {Hearing}. Updating status to failed",
+                    hearingId);
+
+                // Set the booking status to failed as the video api failed
+                //await UpdateFailedBookingStatus(hearingId);
+
+                return Ok(new UpdateBookingStatusResponse {Success = false, Message = errorMessage});
+            }
+        }
+
+        /// <summary>
+        ///     Update the failed hearing status.
+        /// </summary>
+        /// <param name="hearingId">The hearing id</param>
+        /// <returns>Success status</returns>
+        [HttpGet("{hearingId}/update-status")]
```

Comment:
this should be a Put rather than a Get since it modifies data.

think it's probably also worth renaming it so that it's clear it only changes the status to failed.

something like `{hearingId}/failed-status`

`public async Task <IActionResult> UpdateFailedBookingStatus(Guid hearingId)`

Reply:
api changed